week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,64 @@
 // Write your code here
+let images = [
+    { id: "01", 
+      src: "images/01.jpg" },
+    {
+      id: "02",
+      src: "images/02.jpg",
+    },
+    {
+      id: "03",
+      src: "images/03.jpg",
+    },
+    {
+      id: "04",
+      src: "images/04.jpg",
+    },
+  ];
+  let i = 0;
+
+let ImHolder = document.querySelector("#slideBox");
```

Comment:
This variable is highlighted differently because it starts with a capital letter. Remember to keep variables camelCase.